app/scripts/controllers/mmi-controller.js

```diff
@@ -60,6 +60,8 @@ export default class MMIController extends EventEmitter {
       undefined,
       undefined,
       this.securityProviderRequest,
+      // this is should be set now?
+      // getCurrentCaipChainId: ...
```

Comment:
TypeMessageManager got updated to use this. Not sure if it should be included here or not